features/is-analytics-feature/org.wso2.analytics.solutions.is.analytics.feature/src/main/resources/siddhi-files/IS_ANALYTICS_SUSPICIOUS_LOGIN.siddhi

```diff
@@ -156,4 +161,18 @@ update or insert into
         SuspiciousLoginAlertTable
 on      SuspiciousLoginAlertTable.meta_tenantId==meta_tenantId and 
         SuspiciousLoginAlertTable.username==username and  
-    SuspiciousLoginAlertTable.msg==msg;
\ No newline at end of file
+    SuspiciousLoginAlertTable.msg==msg;
+
+-- Retain last one year data in the SecurityAlertTypeTable
+
+from TriggerStream
+select convert(time:dateSub(triggered_time,1,'YEAR'), 'long') as lastYearTimestamp
+insert into LastYearTimestamp;
+
+from TriggerStream join SecurityAlertTypeTable
```

Comment:
Was this delete query verified?
 
from TriggerStream 
delete <Table name> 
on timestamp>=Last yeartimestamp?